src/app/backend/handler/apihandler.go

```diff
@@ -1442,7 +1494,7 @@ func (apiHandler *APIHandler) handleGetPersistentVolumeList(request *restful.Req
 func (apiHandler *APIHandler) handleGetThirdPartyResource(request *restful.Request,
 	response *restful.Response) {
 	dataSelect := parseDataSelectPathParameter(request)
-	result, err := thirdpartyresource.GetThirdPartyResourceList(apiHandler.client, dataSelect)
+	result, err := thirdpartyresource.GetThirdPartyResourceList(request.Attribute("apiclient").(*clientK8s.Clientset), dataSelect)
```

Comment:
Perhaps you could move `request.Attribute("apiclient").(*clientK8s.Clientset)` to a function?